config/nginx/proxy.conf

```diff
@@ -0,0 +1,8 @@
+server {
+  listen 80;
+  listen [::]:80;
+
+  location / {
+    proxy_pass http://example.com;
```

Comment:
Change this to `proxy_pass http://flask_web` (or similar name) when the web application server is set up.